imports/plugins/included/payments-authnet/server/methods/authnet.js

```diff
@@ -91,9 +95,12 @@ Meteor.methods({
     } else {
       throw new Meteor.Error("invalid-transaction-type", "Invalid Transaction Type");
     }
-
-    const result = Promise.await(authResult);
-    return result;
+    try {
+      const result =  Promise.await(authResult);
+      return result;
+    } catch (error) {
+      throw new Meteor.Error("auth-failed", error.message);
```

Comment:
@zenweasel @machikoyasuda we could really use a list of approved error messages, and standardization.. do we have these documented?

Reply:
Added an issue: https://github.com/reactioncommerce/reaction-docs/issues/513